project/TravisOutput.scala

```diff
@@ -0,0 +1,68 @@
+package scala.build
+
+import java.util.TimerTask
+import java.io._
+import java.util.concurrent.TimeUnit
+
+// Prints some output to console periodically if the build itself is silent to avoid Travis CI's 10 minute timeout
+// from aborting the build.
+//
+// A hopefully better behaved version of travis_wait and travis_wait_enhanced.
+object TravisOutput {
+  def install(): Unit = {
+    savedOut = System.out
```

Comment:
The assignment should only be done if the condition below is true, I think

Reply:
Done.